recipes/liblsl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+
+{% set name = "liblsl" %}
+{% set version = "1.14.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sccn/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 196d28608f0c91fb589fd6392e5f4bf2c4d6d5ee4a68ae1df22bf7352a7621b4
+
+build:
+  number: 0
+  run_exports:
+    - liblsl
+
+requirements:
+  build:
+    - cmake >=3.12
+    - {{ compiler('cxx') }}
+  host:
+    - pugixml
```

Comment:
Since this is a static library, you need to either add it's license here or add it as a run dependency.

Reply:
The LICENSE file has a section `This software uses the following libraries:` with the line `pugixml (https://pugixml.org), MIT. pugixml is Copyright (C) 2006-2018 Arseny Kapoulkine.` as suggested by Arseny. From what I have understood, the MIT license doesn't even require attribution in the binary packages, just in the source code.

If it's needed, I can add a notice in the package metadata as well. Do you have an example where I would put it?